help_to_heat/frontdoor/views.py

```diff
@@ -921,8 +921,17 @@ def handle_post(self, request, session_id, page_name, data, is_change_page):
 class SuccessView(PageView):
     def get_context(self, session_id, *args, **kwargs):
         supplier = SupplierConverter(session_id).get_supplier_on_success_page()
+        session_data = interface.api.session.get_session(session_id)
+        benefits = session_data.get("benefits")
+        household_income = session_data.get("household_income")
+        is_eco4_eligible = benefits == "Yes" or household_income == "Less than £31,000 a year"
         referral = portal.models.Referral.objects.get(session_id=session_id)
-        return {"supplier": supplier, "safe_to_cache": True, "referral_id": referral.formatted_referral_id}
+        return {
+            "supplier": supplier,
+            "safe_to_cache": True,
```

Comment:
I think we may not want this now? Given the response now contains data drawn from the user's salary/benefits answer and judging by the comment  where we use `safe_to_cache`. Maybe something to ask for wider opinions about: I'm not sure if we necessarily need to be as strict as we're currently being.

Reply:
considering that this page has always been dynamic (it contains the referral code) it seems it hasn't been safe to cache in some time. this is also the only time in the project this is used so can probably be removed. can ask Sam to confirm?

Reply:
Judging by the comment it's less about the page being dynamic and more about it containing potentially sensitive data. [Looks like](https://beisdigital.atlassian.net/browse/PC-359) it's something that came out of the pen test around go-live last year. Suspect we should stick to that and remove this line. Would keep the other bit of code though.

Reply:
have removed the safe to cache line